### DISCUSSION TITLE
How can I promote my pytest plugin

### DISCUSSION THREAD

**Discussion by zen-xu** [pytest-dev/pytest#9096] [Q&A] [answered]
I write a bdd plugin [pyspock](https://github.com/zen-xu/spock), but I don’t know where to promote it

**Comment by nicoddemus** [comment, marked as answer]
Hi @zen-xu, 

I suggest sending an e-mail to pytest-dev@python.org and testing-in-python@lists.idyll.org, along with sharing in Twitter with the #pytest hashtag.
